game/Submods/NSFW Submod/nsfw_stories.rpy

```diff
@@ -272,63 +272,62 @@ label nsfw_erotic_story_thepen:
+    m 5dkbsa "In my mind, you leaned down to kiss me, and I could almost feel your lips pressed against mine."
+    m 2tkbsb "I felt my body getting hot, and the area down there began to feel a lot more moist than before..."
     m 2gkbsa "I started rubbing myself faster, and faster~"
-    m 2tkbsu "I felt you getting rougher with me as your finger was making it's way up and down my pussy."
+    m 2tkbsu "I imagined that you getting rougher with me as your finger was making its way up and down my pussy."
     m 2tkbsa "You were pressing your tongue against my lips, as if asking permission to enter."
     m 2hubsa "Ehehe~ I was more than willing to give it..."
-    m 2tkbsb "I used my own tongue and forced its way into your mouth."
+    m 2tkbsb "I pictured myself using my own tongue and forcing its way into your mouth."
     m 1hkbssdlb "I don't know why, but I imagined that you were taken aback from my boldness. Ahaha~"
     m 1tkbsb "It was then that I felt myself getting close."
     m 1ekbsb "I kept the pace going, but I could feel my arm starting to get sore from the continuous movement."
-    m 1tkbsa "I ignored it. After all, it was your arm doing the work in my mind~"
-    m 1tsbsa "I got that feeling in my lower-half that told me I was coming..."
-    m 2tsbsb "My arms wrapped around your head and pulled you in as you finished me off."
+    m 1tkbsa "I ignored it. After all, in my mind it was your arm doing the work~"
+    m 1tsbsa "I got that feeling in my lower half that told me I about to come..."
+    m 2tsbsb "I imagined my arms wrapping around your head and pulling you in as you finished me off."
     m 2ekbsb "As I orgasmed, I felt my body jolt from the pleasure."
-    m 3rkbsa "It felt like there was a swirling buildup around my lower-half..."
-    m 3subsb "Which all at once flushed out of my body like fireworks."
-    m 1subsa "It was like an electric pulse was making it's way through my body ever few seconds, jolting me stiff."
+    m 3rkbsa "It felt like there was a swirling buildup in my lower half..."
+    m 3subsb "...Which all at once flushed out of my body like a burst of fireworks."
+    m 1subsa "It was like an electric pulse making its way through my body every few seconds, jolting me stiff."
     m 5eubsa "But it felt incredible..."
-    m 5hkbsb "It took 20 whole seconds before my body calmed down enough and I opened my eyes."
-    m 5gkbsa "My arm flopped to the side, the now drenched pen dropping to the floor."
+    m 5hkbsb "It took twenty whole seconds before my body calmed down enough for me to open my eyes."
+    m 5gkbsa "My arm flopped to the side, the now-drenched pen dropping to the floor."
     m 5hubsb "Ahaha! It was really something, [player]."
     if persistent._nsfw_genitalia == "P":
-        m 5tubsb "I bet that story got {i}you{/i} all stiff too huh, [player]?"
+        m 5tubsb "I bet that story got {i}you{/i} all stiff and aroused too, huh, [player]?"
```

Comment:
I was looking around for examples of how the MAS devs did this, and it seems pretty up in-the-air.
There is 1 example of it being laid out this way: "[sentence here], huh, [player]?"
There are 4 examples of it being laid out this way: "[sentence here] huh, [player]?"
And finally there are 3 examples of being being made like this: "[sentence here], huh [player]?"

Just thought this was interesting in-case we wanted to keep things consistent.

Reply:
Ignore me using laid here, it's a bad habit by the sounds of it lol

Reply:
I think that usage of laid is technically correct, haha. 
Since they use all 3 versions I think any should be fine. Personally I actually like "[sentence], huh [player]?" now that you mention it.

Reply:
Sweet, I'll try to keep my writing consistent with that form of comma placement in the future.